pkg/kubelet/cm/resourceupdates/updates.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourceupdates
+
+// Update is a struct that represents an update to a pod when
+// the resource changes it's status.
+// Later we may need to add fields like container name, resource name, and a new status.
+type Update struct {
+	// PodUID is the UID of the pod which status needs to be updated.
+	PodUID string
+}
```

Comment:
`resourceupdates.Update` feel clunky, but I don't have a clearly better suggestion. Perhaps `resourceupdates.Event`? `resourceupdates.Change` ? can't really say

Reply:
The name Update is used in other places like probes. I feel similarly, but want to keep consistency.

Reply:
works for me.